sourced/ml/transformers/indexer.py

```diff
@@ -0,0 +1,60 @@
+import typing
+from pyspark.rdd import RDD
+from pyspark import Row
+
+from sourced.ml.transformers import Transformer
+
+
+class Indexer(Transformer):
+    """
+    Transformer that creates index mapping for one column in spark rdd.
+    It map all values in the column to its index.
+    One can get python dictionary with mapping from column2id property and unique values list from
+    column_values.
+    """
+    def __init__(self, column: typing.Union[int, str], **kwargs):
+        """
+        :param column: column index or its name in pyspark.RDD for indexing.
+        :param kwargs:
+        """
+        super().__init__(**kwargs)
+        self.column = column
+        self._column2id = None
+        self._column_values = None
+
+    @property
+    def column2id(self):
```

Comment:
How about `__getitem__` instead?

Reply:
One thing, I am not sure it is the best idea. If someone decides to index not string values, or int values, `__getitem__` can have unexpected behavior.

I prefer to add `__getitem__` as a shortcut for `column2id` property. ok?

Reply:
`value_to_index`, yes